### DISCUSSION TITLE
how to get started with GEM reconstruction

### DISCUSSION THREAD

**Discussion by isaacxdiaz** [SysBioChalmers/RAVEN#404] [Help me!] [answered]
Hello,

I am new to all of this and I am trying to use RAVEN for GEM reconstruction. I’ve been consulting the wiki, community issues and other resources available but I can’t seem to get a hand on how it works, what are the commands/language to use, etc. So, I am reaching out hoping there is a place where I can find tutorials and the commands to use? Any information is helpful!

Thank you.

**Comment by edkerk** [comment, marked as answer]
One thing to consider is that RAVEN basically has 2 types of model reconstruction: (a) de novo from KEGG (and MetaCyc); or (b) with the use of a template model and through homology between the two organisms.

We have written a book chapter that guides you through reconstruction of a model for approach (b) (there is also a brief mention of approach (a)), which you can find here: https://www.biorxiv.org/content/10.1101/2021.06.18.448943v1. It comes with a GitHub repository with all the necessary codes: https://github.com/SysBioChalmers/hanpo-GEM. You can use this as a template to start with. Some aspects of the protocol/code is quite specific for the model that you use, so not everything is directly translatable. And it generates a draft model, that would require additional manual curation to make sure it is a high quality model. Nonetheless, it should be able to provide you guidance on how to get started.

If you have any specific questions beyond that, then this Discussion forum is the place to ask! :)
